examples/fe-tools/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@examples/fe-tools",
```

Comment:
nit: i'd probably rename this example to `front-end-tools` or `client-side-tools` just to make it stand out a bit more.